MAINTAINERS.md

```diff
@@ -0,0 +1,5 @@
+# Maintainers
+
+Current active maintainers of this SDK:
+
+- [John Laswell](https://github.com/jlaswell)
```

Comment:
What's the process to get @GrahamCampbell onto this list as well, @duglin ?

Reply:
Per https://github.com/cloudevents/spec/blob/main/docs/SDK-GOVERNANCE.md#new-maintainers you can start a vote to add him.